pytestqt/_tests/test_basics.py

```diff
@@ -80,6 +80,18 @@ def test_widget_kept_as_weakref(qtbot):
     assert widget() is None
 
 
+def test_header(testdir):
+    res = testdir.runpytest()
+    v = get_versions()
+    res.stdout.fnmatch_lines([
+        '*test session starts*',
+        '{qt_api} {qt_api_version} -- Qt runtime {runtime} -- '
+        'Qt compiled {compiled}'.format(qt_api=v.qt_api,
+                                        qt_api_version=v.qt_api_version,
+                                        runtime=v.runtime, compiled=v.compiled)
+    ])

```

Comment:
I had the same concern, but didn't give it much thought... shame on me! :stuck_out_tongue_winking_eye: 

Thanks for the suggestion, I will update the code once I have the change! :smile: